php/php-8.2.0/wlr-build.sh

```diff
@@ -67,6 +70,7 @@ logStatus "Preparing artifacts..."
 mkdir -p ${WLR_OUTPUT}/bin 2>/dev/null || exit 1
 
 # WASMOPD_ASYNCIFY_ARGS="--asyncify --pass-arg=asyncify-ignore-imports"
+WASMOPD_ASYNCIFY_ARGS=-O3
```

Comment:
In purpose?

Reply:
Yes. So that we have some optimization. But you're right that the naming is misleading. Will change

Reply:
Fixed. It is now called `WASM_OPT_ARGS`